Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://hosted.weblate.org/projects/matomo/communityplugin-signup/) for Matomo/CommunityPlugin Signup.



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/matomo/-/communityplugin-signup/horizontal-auto.svg)

Findus23 commented:
I manually added German to see if it was working.
Maybe wait with merging this PR until the other languages also arrive (which might take a while)

alcalyn commented:
Ok. I'm trying to add a spanish translation, but there should have a way to let me join and translate the project, because I cannot translate for now (insuffisant rights)

Findus23 commented:
If you tell me your username, I can add you to the project.
But once we are done migrating, the project will be open for everyone to contribute. It is just locked now so it doesn't interfere with the migration.

alcalyn commented:
On Weblate, I'm also `@alcalyn`, full name is Julien Maulny

Findus23 commented:
This edit from you was unfortunatly lost:
```json
{
  "Signup": {
    "Signup": "Crear una cuenta"
  }
}

alcalyn commented:
Ok it was just a test

Findus23 commented:
If we don't notice anything weird anymore, then these commits should finally be the correct ones (with the correct language codes) and you should be able to merge (or squash and merge) this PR.
Today the migration will be finalized and the project opened to everyone.

weblate committed:
Added translation using Weblate (Vietnamese)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Chinese (Simplified))

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Chinese (Traditional))

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Translated using Weblate (Turkish)

Currently translated at 100.0% (5 of 5 strings)

Translation: Matomo/CommunityPlugin Signup
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-signup/tr/

[ci skip]

Added translation using Weblate (Turkish)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Kaya Zeren <kayazeren@gmail.com>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Thai)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Urdu)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Ukrainian)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Tagalog)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Telugu)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Croatian)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Indonesian)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Hindi)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Hungarian)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Finnish)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Persian)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Basque)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Galician)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Hebrew)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Estonian)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Danish)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Translated using Weblate (Greek)

Currently translated at 100.0% (5 of 5 strings)

Translation: Matomo/CommunityPlugin Signup
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-signup/el/

[ci skip]

Added translation using Weblate (Greek)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Vasilis Lourdas <dev@lourdas.eu>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Spanish (Argentina))

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Czech)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Translated using Weblate (German)

Currently translated at 100.0% (5 of 5 strings)

[ci skip]

Translated using Weblate (German)

Currently translated at 60.0% (3 of 5 strings)

Added translation using Weblate (German)

Co-authored-by: Christoph Ackermann <weblate@ewok-factory.ch>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Lukas Winkler <weblate@lw1.at>
Translate-URL: https://hosted.weblate.org/projects/matomo/communityplugin-signup/de/
Translation: Matomo/CommunityPlugin Signup

weblate committed:
Added translation using Weblate (Belarusian)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Bulgarian)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Catalan)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Arabic)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Amharic)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Bosnian)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Tamil)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Albanian)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Serbian)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Slovenian)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Swedish)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Slovak)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Russian)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Romanian)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Portuguese)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Portuguese (Brazil))

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Polish)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Dutch)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Latvian)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Norwegian Nynorsk)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Norwegian Bokmål)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Japanese)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Italian)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Georgian)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Lithuanian)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Korean)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>

weblate committed:
Added translation using Weblate (Icelandic)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>
